pipeline/data/importers/mono/hplt.py

```diff
@@ -70,33 +72,56 @@ def __init__(self, dataset_path: Path) -> None:
         self.final_lines = CountingStep(
             "How many lines were actually written.",
         )
+        self.filtered_doc_locale = CountingStep(
+            "How many lines were filtered based on document locale.",
+        )
+        self.filtered_line_locale = CountingStep(
+            "How many lines were filtered based on line locales.",
+        )
+        self.filtered_doc_score = CountingStep(
+            "How many lines were filtered based on document scores.",
+        )
+        self.filtered_too_long = CountingStep(
+            "How many lines were filtered based on length.",
+        )
 
     def count_shards_visited(self, *_args):
         self.shards.filtered -= 1
         self.shards.kept += 1
 
 
+def get_hplt_locale(lang_iso6931: str) -> str:
+    """
+    Converts language in ISO-693-1 format to the HPLT format.
+    For example, ru -> rus_Cyrl
+    """
+    locale = icu.Locale(lang_iso6931)
+    # add default script
+    locale = icu.Locale.addLikelySubtags(locale)
```

Comment:
Ok, this doesn't work for Korean... It returns Kore which is a mix of Hang and Hani. I wonder if we should specify the script manually.

Reply:
https://en.wikipedia.org/wiki/ISO_15924#Kore